cores/esp32/HardwareSerial.h

```diff
@@ -86,6 +86,10 @@ class HardwareSerial: public Stream
     {
         return write((uint8_t) n);
     }
+    void setBaudRate(uint32_t baud) 
```

Comment:
don't put the implementation in the header please. just declare here and add to the cpp file